src/functions/sendCandidateResults/application/service/__tests__/template-selector.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { subjectMapper, templateMapper } from '../template-selector';
+import { Language } from '../../../domain/conducted-language';
+import { TestOutcome } from '../../../domain/test-outcome';
+
+const drivingCategories = [
```

Comment:
can this reference be changed to be more meaningful, i know i did it and its in a spec file but my eye keeps getting drawn to it